apps/vast/vast-bridge.py

```diff
@@ -82,6 +84,7 @@ async def start(
     unflatten: bool,
     transform_cmd: str = None,
     sink: str = None,
+    max_open_files: int = 100,
```

Comment:
Could you add a comment for why you chose the value 100?

Reply:
I removed the default value. The default value is already set via our `argparse`.